docs/CONFIG-VARS.md

```diff
@@ -66,14 +66,16 @@ To allow access for other administrative client applications (for example `kubec
 To do this, specify ranges of IP in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).
 Contact your Network System Administrator to find the public CIDR range of your network.
 
-You can use `default_public_access_cidrs` to set a default range for all created resources. To set different ranges for other resources, define the appropriate variable. Use and empty list `[]` to disallow access explicitly.
+Please note that these values are used to add Ingress rules into to an AWS Security Group that by default gets created by terraform. If you specify a pre-existing  [Security Group](#sg_id), you need to add your access CIDRs to that Security Group yourself. The values in this section will not be applied. 
+
+You can use `default_public_access_cidrs` to set a default range for all created resources. To set different ranges for other resources, define the appropriate variable. Use an empty list `[]` to disallow access explicitly.
 
 | <div style="width:50px">Name</div> | <div style="width:150px">Description</div> | <div style="width:50px">Type</div> | <div style="width:75px">Default</div> | <div style="width:150px">Notes</div> |
 | :--- | :--- | :--- | :--- | :--- |
 | default_public_access_cidrs | IP Ranges allowed to access all created cloud resources | list of strings | | Use to to set a default for all Resources |
 | cluster_endpoint_public_access_cidrs | IP Ranges allowed to access the AKS cluster api | list of strings | | for client admin access to the cluster, e.g. with `kubectl` |
-| vm_public_access_cidrs | IP Ranges allowed to access the VMs | list of strings | | opens port 22 for SSH access to the jump and/or nfs VM |
-| postgres_access_cidrs | IP Ranges allowed to access the AWS PostgreSQL Server | list of strings |||
+| <a name="vm_public_access_cidrs">vm_public_access_cidrs</a> | IP Ranges allowed to access the VMs | list of strings | | opens port 22 for SSH access to the jump and/or nfs VM |
```

Comment:
Remove html formatting items from here and line 78, 103

Reply:
html is the only way I found to achieve linking to items in markdown tables. Is there another way?

Reply:
We typically don't have links in md tables. Just reference the section. If you'd like to update every table in all our docs for every provider to enable table item linking, then create a PR for table linking in each provider.

Reply:
note that these anchors are being linked to by docs/BYONetwork.md